apps/financial-aid/web-veita/src/components/History/History.tsx

```diff
@@ -77,11 +77,17 @@ const History = ({ className, applicantName, applicationEvents }: Props) => {
                     </Box>
                   )}
 
-                  {/* TODO if staff sents comment to applicant */}
-                  {/* <Icon icon="checkmark" />{' '}
-                      <Text fontWeight="semiBold">
-                        Skilaboð send á umsækjanda
-                      </Text> */}
+                  {item.eventType === ApplicationEventType.DATANEEDED &&
```

Comment:
We should refactor this into a component that knows what should do with each type, not needed in this PR, but should be a nice refactoring todo 😄